src/main/java/org/jenkinsci/plugins/builduser/varsetter/impl/SCMTriggerCauseDeterminant.java

```diff
@@ -16,7 +16,8 @@ public boolean setJenkinsUserBuildVars(SCMTriggerCause cause,
 			Map<String, String> variables) {
 		
         if (cause != null) {
-			UsernameUtils.setUsernameVars("SCMTrigger", variables);
+			UsernameUtils.setUsernameVars("SCM Change", variables);
```

Comment:
So there will be first and last name then. Maybe `SCMTrigger SCMTrigger` or a dedicated Util method are preferable